worker/src/headers.rs

```diff
@@ -182,10 +183,3 @@ impl From<Headers> for HeaderMap {
         (&headers).into()
     }
 }
-
-impl Clone for Headers {
```

Comment:
We can't use the derive-based clone implementation here as we do an explicit call to `web_sys::Headers::new_with_headers` because the inner JS headers are sometimes immutable. The derive clone will just clone a reference to those potentially immutable headers where-as the explicit clone will perform a deep copy.

Reply:
I won't use workers anymore, feel free to close this PR